client/log.go

```diff
@@ -84,6 +84,17 @@ func (c *Log) TrackMetric(section string, operation bucket.MetricOperation) Clie
 	return c
 }
 
+// TrackMetricWithLabels tracks custom metric with added labels, w/out ok/fail additional sections
+func (c *Log) TrackMetricWithLabels(section string, operation bucket.MetricOperation, labels map[string]string) Client {
+	b := bucket.NewPlain(section, operation, true, c.unicode)
```

Comment:
you could simply replace copy-pasted method body with `return c.TrackMetric(...)`